.github/workflows/integration-tests.yml

```diff
@@ -57,6 +57,44 @@ jobs:
         with:
           prefix: Basic
 
+  run-tests-conda:
```

Comment:
Hmm what do you think the risks of conda breaking are? If we don't expect it to happen a lot I wonder if we should save on costs and move it to periodic.

Reply:
sounds good, it's easier for me to test in-PR here. Will move once all test succeeded